first_hour_on_terra/batch_template.csv

```diff
@@ -0,0 +1,4 @@
+TERRA_USER_EMAIL,POLICY,
```

Comment:
I don't think we should have commas at the end of each line, right? Because that indicates a third column who's values are just empty strings.